app/views/vacancies/index.html.haml

```diff
@@ -23,19 +23,20 @@
             = render partial: 'vacancy', locals: { vacancy: vacancy }
```

Comment:
What are the changes to this view?

Reply:
hmm im not sure there should be any, am checking

Reply:
reverted whatever happened here, wasnt intentional